app/packages/looker/src/elements/three-d.ts

```diff
@@ -89,28 +89,177 @@ export class ThreeDElement extends BaseElement<ThreeDState, HTMLImageElement> {
   }
 
   drawExtension() {
-    this.ctx.globalAlpha = 0.8;
-    this.ctx.font = "34px serif";
-    this.ctx.fillStyle = DEFAULT_FILL_STYLE;
-    this.ctx.textAlign = "center";
-    this.ctx.textBaseline = "middle";
-    this.ctx.fillText(
-      this.isFo3d ? "3D scene" : "point cloud",
-      this.canvas.width / 2,
-      this.canvas.height / 2 - 120
+    // background color: slightly faded dark-ink-black
+    this.ctx.fillStyle = "rgba(17, 25, 40, 0.95)";
+    this.ctx.fillRect(0, 0, this.canvas.width, this.canvas.height);
+
+    this.drawGrid();
+  }
+
+  private drawGrid() {
+    this.ctx.save();
+    this.ctx.globalAlpha = 0.13;
+    this.ctx.strokeStyle = "#60A5FA";
+    this.ctx.lineWidth = 1;
+
+    const width = this.canvas.width;
+    const height = this.canvas.height;
+    const centerX = width / 2;
+    const centerY = height / 2;
+    const gridSize = 40;
+
+    // vertical grid lines
+    for (let x = 0; x <= width; x += gridSize) {
+      this.ctx.beginPath();
+      this.ctx.moveTo(x, 0);
+      this.ctx.lineTo(x, height);
+      this.ctx.stroke();
+    }
+
+    // horizontal grid lines
+    for (let y = 0; y <= height; y += gridSize) {
+      this.ctx.beginPath();
+      this.ctx.moveTo(0, y);
+      this.ctx.lineTo(width, y);
+      this.ctx.stroke();
+    }
+
+    // axes lines (X and Y only)
+    this.ctx.globalAlpha = 0.18;
+    this.ctx.lineWidth = 1.2;
+
+    // X axis (red, horizontal)
+    this.ctx.strokeStyle = "#FF5555";
+    this.ctx.beginPath();
+    this.ctx.moveTo(0, centerY);
+    this.ctx.lineTo(width, centerY);
+    this.ctx.stroke();
+
+    // Y axis (green, vertical)
+    this.ctx.strokeStyle = "#22DD55";
+    this.ctx.beginPath();
+    this.ctx.moveTo(centerX, 0);
+    this.ctx.lineTo(centerX, height);
+    this.ctx.stroke();
+
+    // a small elevated '3D' logo at the top right
+    const logoText = this.isFo3d ? "3D" : "PCD";
+    const logoFont = "bold 18px system-ui, sans-serif";
+    const paddingX = 14;
+    const margin = 10;
+    this.ctx.font = logoFont;
+    const textWidth = this.ctx.measureText(logoText).width;
+    // extra space for cube
+    const rectWidth = textWidth + paddingX * 2 + 32;
+    const rectHeight = 28;
+    const radius = 10;
+    // position at top right
+    const rectX = this.canvas.width - rectWidth - margin;
+    const rectY = margin;
+    // draw background rounded rectangle
+    this.ctx.save();
+    this.ctx.globalAlpha = 0.55;
+    this.ctx.fillStyle = "#151A23";
```

Comment:
same comment

Reply:
same: not quite the right color when you add opacity, and this is a one off case, so i got lazy, but sure